cve_bin_tool/checkers/iptables.py

```diff
@@ -0,0 +1,22 @@
+# Copyright (C) 2022 Orange
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+
+"""
+CVE checker for iptables
+
+https://www.cvedetails.com/product/56953/Netfilter-Iptables.html?vendor_id=17890
+
+"""
+from cve_bin_tool.checkers import Checker
+
+
+class IptablesChecker(Checker):
+    CONTAINS_PATTERNS = []
+    FILENAME_PATTERNS = []
+    VERSION_PATTERNS = [
+        r"([0-9]+\.[0-9]+\.[0-9]+\.?[0-9]*)\r?\niptables",
+        r"iptables-rules>[a-zA-Z %:\n]*([0-9]+\.[0-9]+\.[0-9]+\.?[0-9]*)",
```

Comment:
```suggestion
        r"iptables-rules>[a-zA-Z %:\r?\n]*([0-9]+\.[0-9]+\.[0-9]+\.?[0-9]*)",
```
Tests are failing on windows only.  I'm guessing it's just a missing \r here that's causing the pattern to fail.

Reply:
Fixed